### ISSUE TITLE
Notify users if their account is deleted

### ISSUE THREAD

**Issue by Firefishy** [openstreetmap/openstreetmap-website#5191]
### Problem

When an admin deletes an account there should be an email notification to the user. Accounts are normally deleted due to violation of Terms of Service. eg: "Engage in any form of unsolicited advertising"

### Description

The user should be emailed when an account is deleted / suspended.

Rough draft:
> From: noreply @ osm
> Subject: Notification of OpenStreetMap Account Removal
>
> [Display Name],
>
> We are writing to inform you that your OpenStreetMap account has been removed by an administrator likely due to a violation of our Terms of Use. The Terms of Use prohibit various activities, including but not limited to engaging in any form of unsolicited advertising.
>
> If you believe this decision was made in error, or if you would like to appeal, you may review our Terms of Use [here](https://osmfoundation.org/wiki/Terms_of_Use) and follow the appeal process outlined.
>
> We appreciate your understanding.

**Comment by Firefishy** [comment]
For clarity I'm only suggesting we do this for for accounts "soft" deleted via the website api. Bulk removals are outside of scope.

If we have concerned about emailing dead email addresses then we could decide when we'd send the email. eg: recent account or fresh edits.

**Comment by tomhughes** [comment]
Lots of reasons really...

Firstly I suspect you'll be surprised how many of them will be quite happy to argue the toss - we already get quite a lot that follow up on the suspension email by protesting their innocence and I certainly don't need any more. I hadn't actually noticed that the contact link you gave wasn't the usual support one but still.

Secondly I very much doubt any of them will stop what they're going just because you send them an email - most of them will never even notice as they create a new email account for each client that they probably never look at again and even if they do it's not like they actually care about following rules.

There's also the problem that in many cases we are deleting account from five years ago and we don't really want to remind people that they had a "listing" that has now been removed.

On a detail level you proposed text tells them to follow a link to an appeals process but there is no such process documented at that link that I can see.

**Comment by Firefishy** [comment]
> Firstly I suspect you'll be surprised how many of them will be quite happy to argue the toss - we already get quite a lot that follow up on the suspension message by protesting their innocence and I certainly don't need any more. I hadn't actually noticed that the contact link you gave wasn't the usual support one but still.
> 

I agree. You are already done more than enough. I am happy to seed a group to deal with anyone who challenges the suspension.

> Secondly I very much doubt any of them will stop what they're going just because you send them an email - most of them will never even notice as they create a new email account for each client that they probably never look at again and even if they do it's not like they actually care about following rules.
> 

There appears to be a perceived impression that posting SEO backlinks and "business listing" profiles on OpenStreetMap are acceptable (it isn't). We've been very soft touch in the past. Sending a message when an account is suspended, especially if done soon after the spam has been posted, applies a little pressure and hopefully makes the spammers notice of futile the efforts are.

Examples of OpenStreetMap being promoted as a suitable location to post spammy SEO backlinks or posting "business profiles":
* https://www.youtube.com/watch?v=H2_VtvmfxK4
* https://urionseo.com/how-to-enhance-your-seo-with-openstreetmap-a-step-by-step-backlink-guide/
* https://www.youtube.com/watch?v=HsqXT6T4ZoY
* https://www.youtube.com/watch?v=0ky_ncvXBrs
* https://www.youtube.com/watch?v=R--8KMmy3is

> There's also the problem that in many cases we are deleting account from five years ago and we don't really want to remind people that they had a "listing" that has now been removed.
> 

Agreed. I don't see value in sending a message to an account that hasn't been active in a long period of time. It is very likely after a 180 days the spammer has moved on.

> On a detail level you proposed text tells them to follow a link to an appeals process but there is no such process documented at that link that I can see.

Yes, the draft email needs to revised. It is a starting point.

**Comment by Firefishy** [comment]
@tomhughes @AntonKhorev I kindly ask you to reconsider this issue.

TLDR: I believe that notifying spammers of their account suspension (and content removal) could help reduce spam over time. Please can we run a short test to measure its effectiveness?

Our approach to handling spam has been relatively light, with account removal generally reserved to reported profile spam. However, OpenStreetMap.org appears to be increasingly targeted by [unethical SEO companies](https://urionseo.com/how-to-enhance-your-seo-with-openstreetmap-a-step-by-step-backlink-guide), as a place to post "backlink" spam. Google classifies this behaviour as "Site Reputation abuse."

Sampling a small set of account sign-ups, it appears that approximately 30% to 40% appeared to be spam accounts, created solely to post profile spam. The volume seems to be rising.

This spam imposes several costs on us, including:
* Wasting database resources
* Wasting valuable administrative time dealing with the spam and reports
* Damaging our online reputation in search results
* Potential legal risks, as some spam links are illegal, such as gambling sites, fraud, or even claims of sites offering child sexual abuse material.

I’ve spent several hours each day in recent weeks suspending recent profile spammers. However, I believe that simply notifying these spammers that their accounts (and content) has been suspended could serve as a deterrent, discouraging further spam activity.

I'm in no doubt that I've suspended a few legitimate mappers' accounts by mistake. In one instance a mapper was able to reach me, and I re-enabled their account. We should offer a basic mechanism for a legitimate mapper to report a mistake. Related see: https://github.com/openstreetmap/openstreetmap-website/issues/5097

Proposal:

I suggest a small experiment to measure the impact of account suspension notifications:

1. For one week, we suspend spam accounts as usual, without sending notifications.
2. For the following week, we suspend spam accounts but send a notification email to the users, informing them of the suspension. I’m happy to set up a dedicated email address to handle any responses to these notifications.

At the end of the two weeks, we can compare the amount of spam posted and evaluate whether notifications had any effect.

I believe this is just one small step in a broader set of anti-spam measures. We'll needs other mechanisms to fight spam effectively, and I see this as an incremental improvement that can help guide us toward stronger protections over time.

**Comment by tomhughes** [comment]
An unethical SEO company isn't going to care one iota if it gets a nastygram from us though?

**Comment by AntonKhorev** [comment]
How to put spam into lat/lon: https://youtu.be/0ky_ncvXBrs?si=uJsAZ5W6RuBabNuG&t=195

**Comment by Firefishy** [comment]
> An unethical SEO company isn't going to care one iota if it gets a nastygram from us though?

They do care if their spam is posted. If it removed soon after they posted it then their efforts have been in vain. Unless they check back there has been no feedback loop.
